src/controllers/user.controller.js

```diff
@@ -225,6 +227,64 @@ class UserController {
     }
   }
 
+  /**
+   *
+   *
+   * @description This is the method that generates the password reset email
+   * @param  {object} req The request object
+   * @param  {object} res The response object
+   * @returns {object} json response
+   */
+
+  /**
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} - response object
+   * @memberof UserController
+   */
+  static async resetPassword(req, res) {
+    try {
+      const user = await BaseRepository.findOne(User, {
+        where: { email: req.body.email }
+      });
+      if (!user) {
+        return res.status(401).json({
+          message: 'Invalid credentials supplied'
```

Comment:
Can you use a better message here? You can try something like this "No registered user with this email address"

Reply:
from what the TTL said I can't inform the user that the email is not registered but to find a way to inform him about an invalid credentials.